File: 3870-count-commas-in-range/3870-count-commas-in-range.java
class Solution {
    public int countCommas(int n) {
        if(n<1000){
            return 0;
        }
        // else if(n>=1000 && n<10000){
        //     int c=(n+1)-1000;
        //     return c;
        // }
        // else if(n>=10000 && n<100000){
        //     int c=(n+1)-1000;
        //     return c;
        // }
        else{
            return (n+1)-1000;
        }
    }
}

File: 3870-count-commas-in-range/3870-count-commas-in-range.py
class Solution:
    def countCommas(self, n: int) -> int:
        if(n<1000):
            return 0
        elif(n>=1000 and n<10000):
            c=(n+1)-1000
            return c
        elif(n>=10000 and n<100000):
            c=(n+1)-1000
            return c
        else:
            return (n+1)-1000
        